create_infotexts.py

```diff
@@ -121,19 +121,19 @@ def generate_infobox_template(item, img, places_mapping):
     if item["Beskrivning"]:
         infobox += img.data["enriched_description"]
     else:
-        infobox += "Svenska Cypernexpeditionen 1927-1931"  # Generates six cases only
+        infobox += "Svenska Cypernexpeditionen 1927–1931"  # Generates six cases only
         img.meta_cats.append("Media_contributed_by_SMVK_with_poor_description")
 
     infobox += "}}\n"
-    infobox += "{{en|The Swedish Cyprus expedition 1927-1931}}"
+    infobox += "{{en|The Swedish Cyprus expedition 1927–1931}}"
     infobox += "\n"
 
     infobox += "| depicted people    = " + img.data["depicted_people"] + "\n"
 
     infobox += "| depicted place     = " + img.data["depicted_place"] + "\n"
 
     infobox += "| date               = "
-    if not item["Fotodatum"] or item["Fotodatum"] == "1927-1931":
+    if not item["Fotodatum"] or item["Fotodatum"] == "1927–1931":
```

Comment:
Does SMVK also use the long dash in their data? otherwise this should still be a normal one?